service/api/service.go

```diff
@@ -683,6 +684,18 @@ func (s *serviceImpl) ApiV1WorkflowDumpPost(ctx context.Context, request iwfidl.
 	}, nil
 }
 
+func (s *serviceImpl) ApiInfoHealth(ctx context.Context) *iwfidl.HealthInfo {
+	hostName, err := os.Hostname()
+	if err != nil {
```

Comment:
Let’s swallow this error here and put a default message like host name not available. Because it’s not worth returning error here

Reply:
Ohh sorry i misread it on my phone